analysis/topEFT/update_json_sow.py

```diff
@@ -28,10 +28,50 @@
 #   same name, but in different sub-directories, then it skips updating that file and instead prints
 #   an error displaying the confounding JSONs.
 
-WEIGHTS_NAME = 'SumOfWeights'
-JSON_KEY_NAME = 'nSumOfWeights'
 MAX_PDIFF = 1e-7
 
+WEIGHTS_NAME_LST = [
+    'nom',
+    'ISRUp',
+    'ISRDown',
+    'FSRUp',
+    'FSRDown',
+    'renormUp',
+    'renormDown',
+    'factUp',
+    'factDown',
+    'renormfactUp',
+    'renormfactDown',
+]
+
+# Construct a dict to hold the hist name and json name, format:
+# d = {
+#   'varUp': {
+#       'hist_name': 'SumOfWeights_varUp',
+#       'jsn_key_name': 'nSumOfWeights_varUp',
+#   }
+# }
```

Comment:
Can we remove this commented out code?

Reply:
This comment is just supposed to explain the naming convention being used in the dictionary that it is constructing (since it might not be obvious otherwise). However, if you'd have a suggestion about how to express this in a more useful way I'd of course be happy to modify the comment.

Reply:
Oh sorry, I guess I scrolled to fast and thought it was actual code. This is actually a good thing to keep in!

Reply:
Also, just to try to clarify, this is not actually commented out code, it's really just pseudo code that's supposed to explain for the structure of the dictionary that the function creates.